scenarios/aws/fakeintake/fakeintake.go

```diff
@@ -60,6 +60,7 @@ func NewECSFargateInstance(e aws.Environment, name string, option ...Option) (*f
 			map[string]awsxEcs.TaskDefinitionContainerDefinitionArgs{"fakeintake": *fargateLinuxContainerDefinition(params.ImageURL, apiKeyParam.Name, params.Memory-600, params.DDDevForwarding)},
 			apiKeyParam.Name,
 			nil,
+			true,
```

Comment:
If we change `useLatestStableAgent` is changed to `agentImage` we should pass `public.ecr.aws/datadog/agent` here